docs/04-api-reference/front-commerce-core/makeCustomizableData.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "makeCustomizableData"
+sidebar_position: 7
+description:
+  "The `makeCustomizableData` function allows you to execute a series of
+  transformations on an objet."
+---
+
+<p>{frontMatter.description}</p>
+
+## `makeCustomizableData` function
+
+### Params
+
+| Name                | Type    | Description                                         |
+| ------------------- | ------- | --------------------------------------------------- |
+| `initialTransforms` | `any[]` | Array of [transform functions](#transform-function) |
+
+#### Example
+
+```ts
+import { makeCustomizableData } from "@front-commerce/core/graphql";
+
+const transformer = (data) => {
+  // Do your transformation
+  return data;
+};
+
+const customData = makeCustomizableData([transformer]);
+```
+
+### `registerTransform`
```

Comment:
**nit:** There is no difference (in presentation) between `makeCustomizableData`'s parameters and return values, which makes it confusing a bit (we can't really know where `registerTransform` is coming from without looking at the example here).

**suggestion:** Add a "Parameters" / "Returns" to each method's reference, so that readers know what to pass and what to expect in return.

Reply:
I've added what i think can fix the issue, can you check it and tell me if you think it's better now ?

Reply:
Yep, looks good, thanks !